src/nccl_ofi_net.c

```diff
@@ -992,9 +992,17 @@ static ncclResult_t ofi_init(ncclDebugLogger_t logFunction)
 	NCCL_OFI_INFO(NCCL_INIT | NCCL_NET, "Using " PACKAGE_STRING);
 
 	/*
-	 * RDMAV_FORK_SAFE environment variable makes the rdma-core
-	 * library fork-safe. This significantly increases cost of memory
-	 * registration when huge pages are enabled.
+	 * FI_EFA_FORK_SAFE environment variable tells Libfabric to enable
+	 * fork-safe support in legacy versions of the rdma-core library.
+	 * This significantly increases cost of memory registration. Libfabric
```

Comment:
How does this increase memory registration cost when huge pages get disabled inside libfabric?

Reply:
This was just copied from the earlier version of the comment.  I removed this sentence.